.github/workflows/linux.yml

```diff
@@ -903,6 +898,14 @@ jobs:
             python3 -m pip install $ov_dev_wheel_name[mxnet,caffe,kaldi,onnx,tensorflow2,pytorch]
           popd
 
+      - name: Install Python API tests dependencies
+        run: |
+          # For torchvision to OpenVINO preprocessing converter
+          python3 -m pip install -r ${INSTALL_TEST_DIR}/python/preprocess/torchvision/requirements.txt
+
+          # TODO: replace with Python API tests requirements
```

Comment:
should we apply this `TODO`?

Reply:
I tried, the Python API tests requirements brakes mo tests, I would suggest doing that in separate PR (as part of python deps refactoring)